pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -87,6 +88,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return fmt.Errorf("intrusiondetection-controller failed to watch installer job: %v", err)
 	}
 
+	// Watch for changes to to primary resource LogCollector, to determine if syslog forwarding is
+	// turned on or off.
+	// Since this syslog forwarding is optional, we don't need to return an error if LogCollector
+	// is not found.
```

Comment:
@tmjd not sure about this part. 

Syslog forwarding will only be set if this is a management or standalone cluster, so I suppose I could try to get a `managementClusterConnection` resource, if I find one then it must be managed cluster and I don't need to return an error (proceed as if nothing bad happened). 

Of course, the other angle to this is even for a management or standalone cluster, syslog forwarding is optional. So if it cannot watch a LogCollector then perhaps it should proceed as if nothing bad happened regardless? 

That's why I have it like this right now.